docs/guide/apm-release-notes.asciidoc

```diff
@@ -25,7 +31,24 @@ For a full list of changes, see the
 
 *APM Server*
 
-* Support for {apm-server-ref-v}/ilm.html[index lifecycle management (ILM)]
+* Added support for {apm-server-ref-v}/ilm.html[index lifecycle management (ILM)]:
+ILM enables you to automate how you want to manage your indices over time,
```

Comment:
> enables .. how you want to...

The "how you want" part is not true I think because in first version we are hardcoding the policies, iirc (@simitt correct me if im wrong)

Reply:
correct, nothing is configurable in the server atm